<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
    background-color: transparent;
};</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="practice_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
          </property>
          <property name="text">
           <string>Практика</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="diary_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
          </property>
          <property name="text">
           <string>Обучение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="requests_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
          </property>
          <property name="text">
           <string>Заявки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="homepage_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
          </property>
          <property name="text">
           <string>Личный кабинет</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
